fdbserver/workloads/ConflictRange.actor.cpp

```diff
@@ -328,7 +328,8 @@ struct ConflictRangeWorkload : TestWorkload {
 
 					if (res.size() == originalResults.size()) {
 						for (int i = 0; i < res.size(); i++) {
-							if (res[i] != originalResults[i]) {
+							if (res[i] != originalResults[i] && !res[i].key.startsWith("\xff"_sr) &&
+							    !originalResults[i].key.startsWith("\xff"_sr)) {
```

Comment:
I just realized I wrote this slightly wrong. What I intended was this:

```
if (res[i] != originalResults[i] && !(res[i].key.startsWith("\xff"_sr) &&
 							 originalResults[i].key.startsWith("\xff"_sr))) {
```

That is to say, this should only decline to do the comparison if both keys start with `\xff`. If only one key does, then that would indicate something went wrong.

The current version is ok in that it doesn't fail, but it is slightly weaker.

Reply:
I can apply this change across all 3 branches (main, 7.1, 7.0) if you'd like, and then add you as a reviewer

Reply:
Sounds good, thank you